### DISCUSSION TITLE
C99 extension on windows

### DISCUSSION THREAD

**Discussion by patrick-nicodemus** [pypa/cibuildwheel#2113] [Q&A]
I want to use a C99 extension with Python. The code uses variable length arrays which are not part of the C++ language. MSVC does not support this C99 feature and it doesn't look like they plan to.
Can I use cibuildwheel to cross-compile my code on Windows, perhaps with Clang instead of MSVC? Or am I out of luck?

**Comment by Czaki** [comment]
If I correctly remember, the clang on Windows should produce binary compatible with MSVC, so it may be worth trying. Hoverer, you are the first coming with such a problem (or first that I spot) so I cannot point you to any instructions for setup clang on action runner and which env variable need to be set. 

(As I know, the `gcc` is not compatible with MSVC on Windows, so only clang left to check).
